### ISSUE TITLE
sharded / parallel tests and sort order

### ISSUE THREAD

**Issue by FDIM** [Evilweed/protractor-beautiful-reporter#44]
First of all thanks for the reporter!

I think I've found a nice way to guarantee correct sort order even with `shardTestFiles` set to true. It was still a bit off with the one provided in the readme file. 

What I do is patch jasmine2MetaDataBuilder and add instanceId to each spec which happens to be process.pid:

```
var org = { jasmine2MetaDataBuilder: htmlReporter.jasmine2MetaDataBuilder };
htmlReporter.jasmine2MetaDataBuilder = function () {
    var metadata = org.jasmine2MetaDataBuilder.apply(this, arguments);
    metadata.instanceId = process.pid;
    return metadata;
};
```

Sort function:
```
module.exports = function sort(a, b) {

    var valueA = a.instanceId * a.timestamp;
    var valueB = b.instanceId * b.timestamp;

    if (valueA < valueB) {
        return -1;
    }
    return 1;
}
```

It would be nice if something like this could be incorporated.

**Comment by Evilweed** [comment]
@FDIM Might be good idea to make this change in **jasmine2MetaDataBuilder** permanent (available for everyone without patching).

**Comment by Evilweed** [comment]
@FDIM I've found one issue (You were using sessionId name in one place and instanceId in other), fixed it, and updated base sort function to take into account instanceId.

Released as v1.2.0.
Thanks for PR!
Cheers! :)
